src/shuffle.h

```diff
@@ -57,6 +66,7 @@ class ShuffleChain {
     std::vector<ShuffleItem> _items;
     std::deque<size_t> _window;
     std::deque<size_t> _pool;
+    std::mt19937 _rnd_generator;
```

Comment:
This one seems kinda odd as well. If we're going to shorten `_random_generator`, might as well just do `_rng`. RNG is a well-known acronym anyways.

Reply:
I'll go with `_rng`.